src/MF_Modules/MFEEPROM.cpp

```diff
@@ -8,41 +8,47 @@
 #include "MFEEPROM.h"
 #include <EEPROM.h>
 
-MFEEPROM::MFEEPROM()
+MFEEPROM::MFEEPROM() {}
+
+void MFEEPROM::init(void)
 {
-    eepromLength = EEPROM.length();
+    _eepromLength = EEPROM.length();
 }
 
 uint16_t MFEEPROM::get_length(void)
 {
-    return eepromLength;
+    return _eepromLength;
 }
 
-void MFEEPROM::read_block(uint16_t adr, char data[], uint16_t len)
+bool MFEEPROM::read_block(uint16_t adr, char data[], uint16_t len)
 {
+    if (adr + len > _eepromLength) return false;
```

Comment:
is the check > correct, in other locations we are using >=

I think it should be >=

Reply:
That was addressed in [@elral's comment above](https://github.com/MobiFlight/MobiFlight-FirmwareSource/pull/187#issuecomment-1149908734), he was right in spotting the >/>= issue.